.gitignore

```diff
@@ -0,0 +1,9 @@
+*.doctree 
```

Comment:
We always use a `.gitignore` to keep files like `.DS_Store` and other editing cruft out of the repo.